antora.yml

```diff
@@ -1,10 +1,10 @@
 # Uncomment name and title to build for SUSE Manager
-name: suse-manager
-title: SUSE Manager
+#name: suse-manager
```

Comment:
This should go on a separate URL, but... OK

Reply:
Didn't we agree to have a separe PR for this?

Reply:
Well, since I am not getting a reply, I am asking this again: why this is this part of the PR?

How do you plan to backport this to `Manager-4.2` branch? You should squash this PR so you cherrypick only one commit, which means you don't need this.

Otherwise you have 29 commits already, that you need to cherrypick, with the exception of one for `antora.yaml`, assuming that it doesn't include anything else :-)